Packs/DeepL/Playbooks/playbook-DeepL_Translate_Document.yml

```diff
@@ -0,0 +1,382 @@
+        no matter how many characters are included in the document.  Because the request
+        includes a file upload, it must be an HTTP POST request containing multipart/form-data.
+        This call returns immediately after the document was uploaded and queued for
+        translation. Further requests must be sent to the API to get updates on the
+        translation progress or to download the translated document once the translation
+        is finished (see other document request types below).  Once the document is
+        fully uploaded, the translation starts immediately. Please be aware that the
+        uploaded document is automatically removed from the server after the translation
+        is done. You have to upload the document again in order to restart the translation.  The
+        maximum upload limit for any document is 10MB and 1.000.000 characters.
+      id: ba640463-d64c-4595-8432-8039b8d46da9
+      iscommand: true
+      name: DeepL Submit Document
+      script: '|||deepl-submit-document'
+      type: regular
+      version: -1
+    taskid: ba640463-d64c-4595-8432-8039b8d46da9
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 195
+        }
+      }
+  "4":
+    id: "4"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: c3e62761-de62-4fdb-828b-dc5e672baa37
+      iscommand: false
+      name: Done
+      type: title
+      version: -1
+      description: ''
+    taskid: c3e62761-de62-4fdb-828b-dc5e672baa37
+    timertriggers: []
+    type: title
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 1070
+        }
+      }
+  "5":
+    id: "5"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "6"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      document_id:
+        complex:
+          accessor: document_id
+          root: DeepL.DocumentSubmission
+          transformers:
+          - operator: LastArrayElement
+      document_key:
+        complex:
+          accessor: document_key
+          root: DeepL.DocumentSubmission
+          transformers:
+          - operator: LastArrayElement
+      filename:
+        complex:
+          filters:
+          - - left:
+                iscontext: true
+                value:
+                  simple: inputs.translated_filename
+              operator: isNotEmpty
+          root: inputs.translated_filename
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      description: Get the translated document
+      id: 2173539b-29c4-4603-8320-4659ad6cc997
+      iscommand: true
+      name: Get Translated Document
+      script: '|||deepl-get-document'
```

Comment:
@Hruuttila I believe it's worth adding the output path for this command since it is currently empty:
![image](https://user-images.githubusercontent.com/81556849/161951650-fd1507bc-9ceb-40b7-9c06-539aac65d1b1.png)

Reply:
Actually, the standard practice should be to return the output to "InfoFile" (in your case since it is not a file for analysis):
https://xsoar.pan.dev/docs/integrations/context-and-outputs#return-info-file

So under the "InfoFile" you would put all of the file's info, such as name and EntryID (which I saw you are setting it later).

Reply:
@tomer-pan Yeah I changed it to the InfoFile. thx. I dont understand what I should put into the output path as the command returns the file into the war-room and it goes under the File key?

Reply:
Maybe @bziser can assist with that?